.gitignore

```diff
@@ -16,7 +16,7 @@
 ###> symfony/web-server-bundle ###
 /.web-server-pid
 ###< symfony/web-server-bundle ###
-
+docker-compose.yml
 /docker-compose.yml/
```

Comment:
This line should be replaced with your one.

Reply:
yeah